features/submit_filter.feature

```diff
@@ -180,9 +180,10 @@ Feature: Submit Filter Private Endpoints Not Enabled
           "is_area_type": true
         }
       ],
-      "type": "flexible"
+      "type": "flexible",
+      "population_type": "Example"
     }
     """
     And the following Export Start events are produced:
-      | InstanceID        | DatasetID            | Edition          | Version | FilterOutputID                       |
-      | TEST-INSTANCE-ID  | cantabular-example-1 | 2021             | 1       | 94310d8d-72d6-492a-bc30-27584627edb1 |
+      | InstanceID        | DatasetID            | Edition          | Version | FilterOutputID                       | DimensionIDs |
+      | TEST-INSTANCE-ID  | cantabular-example-1 | 2021             | 1       | 94310d8d-72d6-492a-bc30-27584627edb1 | Number of siblings (3 mappings),City|
```

Comment:
`Number Of siblings (3 mappings)` and `City` are lables not ids. The ids would be `siblings_3` and `city` probably best to be accurate here

Reply:
resolved